### PR TITLE
Fix grafana dashboard to handle wildcard scaledObject variables

### LINKED ISSUE #6214: Grafana dashboard doesn't handle wildcard scaledObject variables properly for some graphs
### Report

All 4 of the ScaleTarget graphs won't render graphs if more than one scaledObject exists in the namespace and the "All" option is selected for the variable. This is due to incorrect wildcard matching definition in the graphs for scaledObject variable.

### Expected Behavior

Expect graphs to display metrics for all scaledObjects

### Actual Behavior

No metrics are displayed

### Steps to Reproduce the Problem

1. View a namespace that has multiple scaledObjects
2. Select the "All" value for scaledObject variable

### Logs from KEDA operator

N/A

### KEDA Version

2.15.1

### Kubernetes Version

1.29

### Platform

Amazon Web Services

### Scaler Details

metricsApiScaler

### Anything else?

I have a PR ready to submit

### PR DESCRIPTION
**Pull request by alexkuretz**
_Provide a description of what has been changed_

Grafana dashboard now properly handles wildcard scaledObject variables in graphs

### Checklist

- [x] I have verified that my change is according to the [deprecations & breaking changes policy](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
- [x] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Fixes #6214

### CODE REVIEW

**Review by JorTurFer — COMMENTED** [review]
LGTM! Could you update the changelog with the issue? 🙏

**Comment by alexkuretz** [comment]
CHANGELOG updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ Here is an overview of all new **experimental** features:
 - **GitHub Scaler**: Fixed pagination, fetching repository list ([#5738](https://github.com/kedacore/keda/issues/5738))
 - **Kafka**: Fix logic to scale to zero on invalid offset even with earliest offsetResetPolicy ([#5689](https://github.com/kedacore/keda/issues/5689))
 - **RabbitMQ Scaler**: Add connection name for AMQP ([#5958](https://github.com/kedacore/keda/issues/5958))
+- **Grafana dashboard**: Fix dashboard to handle wildcard scaledObject variables ([#6214](https://github.com/kedacore/keda/issues/6214))
```
Comment (JorTurFer) [inline CHANGELOG.md:77]: Items are ordered alphabetically (that's why the CI check fails), could you fix it?

**Review by JorTurFer — APPROVED** [review]
Thanks a lot!

**Pull request merged**